### PR TITLE
Add podcast to Course page

### PR DESCRIPTION
**Pull request by zacjones93**
This adds podcasts to the course page*.

*We aren't sending the simplecast guid over graphql, I'm not sure how to do this but once we do that Podcasts will start showing up

Build a Corgi Up-boop Web App with Netlify Serverless Functions and Ha

### CODE REVIEW

File: src/components/layouts/collection-page-layout.tsx [hunk 1]
```diff
@@ -421,7 +425,7 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
                 <PlayButton lesson={nextLesson} />
               </div>
 
-              <Markdown className="prose dark:prose-dark md:prose-lg md:dark:prose-lg-dark text-gray-900 dark:text-gray-100 mt-6">
+              <Markdown className="prose dark:prose-dark md:prose-lg md:dark:prose-lg-dark text-gray-900 dark:text-gray-100 mt-6 mb-6">
```
Comment (zacjones93) [inline src/components/layouts/collection-page-layout.tsx:428]: add a little spacing for podcast

File: src/components/layouts/collection-page-layout.tsx [hunk 2]
```diff
@@ -442,6 +446,9 @@ const CollectionPageLayout: React.FunctionComponent<CoursePageLayoutProps> = ({
                   </div>
                 )}
               </div>
+              {!isEmpty(podcast) && (
```
Comment (zacjones93) [inline src/components/layouts/collection-page-layout.tsx:449]: Check to make sure that podcast is actually set

File: src/components/layouts/collection-page-layout.tsx [hunk 3]
```diff
@@ -736,4 +743,47 @@ const Fresh = ({freshness}: {freshness: any}) => {
...
+            </span>
+          )}
+        </h3>
+        <iframe
+          height="52px"
+          width="100%"
+          frameBorder="no"
+          scrolling="no"
+          seamless
+          src={`https://player.simplecast.com/${id}?dark=false`}
```
Comment (zacjones93) [inline src/components/layouts/collection-page-layout.tsx:777]: darkmode always turned off but I tried it out and it's set as black so it looks weird anyway.

File: src/components/layouts/collection-page-layout.tsx [hunk 4]
```diff
@@ -736,4 +743,47 @@ const Fresh = ({freshness}: {freshness: any}) => {
   )
 }
 
+const CoursePodcast = ({
```
Comment (zacjones93) [inline src/components/layouts/collection-page-layout.tsx:746]: taken from <CoursePodcast /> in rails

File: src/components/layouts/collection-page-layout.tsx (hunk 3 above)
Comment (vojtaholik) [reply src/components/layouts/collection-page-layout.tsx:777]: you can specify color with `&color=111827` and check for color mode with `{useTheme} from 'next-themes'`

Comment (zacjones93) [reply src/components/layouts/collection-page-layout.tsx:777]: oh thanks @vojtaholik, I'll check that out

**Review by joelhooks — APPROVED** [review]

**Pull request merged**